src/stylesheets/component/_mainheader.scss

```diff
@@ -0,0 +1,63 @@
+@use '../abstract/' as * ;
+.main-header{
+    width: 1043px;
+    height: 880px;
```

Comment:
Don't fix width and height in pixels that will make it difficult for you when you're responsive.